source/extensions/filters/http/json_to_metadata/filter.cc

```diff
@@ -100,6 +103,16 @@ Rules FilterConfig::generateRequestRules(
   return rules;
 }
 
+Rules FilterConfig::generateResponseRules(
+    const envoy::extensions::filters::http::json_to_metadata::v3::JsonToMetadata& proto_config)
```

Comment:
ideally we want to DRY here. e.g., 
`generateRules(const envoy::extensions::filters::http::json_to_metadata::v3::Rules&)` here 
and construct `request_rules_(generateRules(proto_config.response_rules().rules()))` in constructor

same thing with allow content type and allow empty type

Reply:
> ideally we want to DRY here. e.g., `generateRules(const envoy::extensions::filters::http::json_to_metadata::v3::Rules&)` here and construct `request_rules_(generateRules(proto_config.response_rules().rules()))` in constructor
> 
> same thing with allow content type and allow empty type

might optimize this in the future when contribute to upstream